components/textbox/index.ts

```diff
@@ -33,8 +33,8 @@ const makeMapStateToProps = () => {
         return {
             currentUserId: getCurrentUserId(state),
             currentTeamId: teamId,
-            profilesInChannel: getProfilesInChannel(state, ownProps.channelId, true),
-            profilesNotInChannel: getProfilesNotInChannel(state, ownProps.channelId, true),
+            profilesInChannel: getProfilesInChannel(state, ownProps.channelId, {active: true}),
+            profilesNotInChannel: getProfilesNotInChannel(state, ownProps.channelId, {active: true}),
```

Comment:
I modified the signature of these two functions on redux to make the last param a `Filters` object instead of an `active` boolean